File: CoffeeShopProject/Models/ThucDon.cs
using System;
using System.Collections.Generic;

namespace CoffeeShopProject.Models
{
    public partial class ThucDon
    {

        public int MaThucDon { get; set; }
        public string TenThucDon { get; set; }
        public string HinhAnh { get; set; }
        public int? MaLoai { get; set; }
        public double? Gia { get; set; }
        public int? KhuyenMai { get; set; }
        public string MoTa { get; set; }
        public double? GiaKhuyenMai => (Gia - Gia * KhuyenMai / 100);
    }
}

File: CoffeeShopDATN_Android/app/src/main/java/fpo/thanh/coffeeshop/domain/Room/RoomModel/ThucDon.java
package fpo.thanh.coffeeshop.domain.Room.RoomModel;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

@Entity
public class ThucDon {

    @PrimaryKey(autoGenerate = false)
    public Integer maThucDon;
    @ColumnInfo(name ="tenLoai")
    public String tenLoai;
    @ColumnInfo(name ="getGiaKhuyenMai")
    public Double getGiaKhuyenMai;
    @ColumnInfo(name ="tenThucDon")
    public String tenThucDon;
    @ColumnInfo(name ="hinhAnh")
    public String hinhAnh;
    @ColumnInfo(name ="maLoai")
    public Integer maLoai;
    @ColumnInfo(name ="gia")
    public Double gia;
    @ColumnInfo(name ="khuyenMai")
    public Integer khuyenMai;
    @ColumnInfo(name ="moTa")
    public String moTa;
    @ColumnInfo(name ="giaKhuyenMai")
    public Double giaKhuyenMai;
}
